docs/install/minikube-installation-guide.md

```diff
@@ -126,10 +126,10 @@ Containers components to help with this, and then use `kubectl` on the host (tha
 configured for you) to deploy them:
 
 ```sh
-$ git clone https://github.com/kata-containers/packaging.git
-$ cd packaging/kata-deploy
-$ kubectl apply -f kata-rbac.yaml
-$ kubectl apply -f kata-deploy.yaml
+$ git clone https://github.com/kata-containers/kata-containers.git
+$ cd kata-containers/tools/packaging/kata-deploy
+$ kubectl apply -f kata-rbac/base/kata-rbac.yaml
+$ kubectl apply -f kata-deploy/base/kata-deploy.yaml
```

Comment:
Looks like these 2 lines need to be back-ported to https://github.com/kata-containers/documentation/.

Reply:
yes, the 1.x doc project has the same error. Thank you @jodh-intel. I am sorry just force on 2.0-dev.